suite-native/navigation/src/components/Screen.tsx

```diff
@@ -9,17 +9,19 @@ import { BottomTabBarHeightContext } from '@react-navigation/bottom-tabs';
 
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 import { Color, nativeSpacings } from '@trezor/theme';
-import { Box, Divider } from '@suite-native/atoms';
 import { selectIsAnyBannerMessageActive } from '@suite-common/message-system';
+import { Box } from '@suite-native/atoms';
 
-import { ScreenContent } from './ScreenContent';
+import { ScreenContentWrapper } from './ScreenContentWrapper';
+import { ScreenHeader } from './ScreenHeader';
 
 type ScreenProps = {
     children: ReactNode;
-    header?: ReactNode;
     footer?: ReactNode;
-    hasDivider?: boolean;
+    subheader?: ReactNode;
+    customHeader?: ReactNode;
     hasStatusBar?: boolean;
+    shouldDisplayHeader?: boolean;
```

Comment:
nit: Why not `isHeaderDisplayed`?

Reply:
I think both are fine and we use both of them throughout the code so wouldn't change it now.

Reply:
https://github.com/trezor/trezor-suite/pull/9629/commits/03a235d881d06bd6db5a113ed5644a62f6b7a749